### PR TITLE
Baremetal new guides (continued)

### PR DESCRIPTION
**Pull request by GingerGeek**
Hi, this is just a continuation of my previous PR #498 with guides for OKD deployments on bare metals as the last one got merged and I still had guides to write!

Apologies that it's taking me a while to sit down and write all of these, lots of time pressures at the moment!

Feedback and suggestions welcome :)

### CODE REVIEW

**Comment by GingerGeek** [comment]
I've rebased this against master and have completed the custom node IP guide. Sorry in advanced for the hacks I've done :)

File: Guides/UPI/baremetal/nodes-custom-ip.md [hunk 1]
```diff
@@ -0,0 +1,273 @@
+Setting Custom Node IPs
+---
+
+When deploying in non-standard network environments you may need to control what IPs are used by OKD and K8s services to ensure traffic flows over the correct networks/interfaces.
+
+Kubelet and other services used in the bootstrap don't use DNS to resolve IP addresses. An IP is either provided to the Kubelet command or one will be automatically discovered.
+
+By default OKD and K8s services often attempt to auto discover the IP to use based on the routability of each address. In a situation where you have multiple interfaces with valid routes, then the first valid IP will be used by interface order. This means in a system setup where you have `eno1` (routed public network) and `eno2` (routed private network), then the address on the `eno1` network will be used.
+
+> You can look at the runtime resolver for Node IPs [here](https://github.com/openshift/baremetal-runtimecfg/blob/master/cmd/runtimecfg/node-ip.go)
```
Comment (vrutkovs) [inline Guides/UPI/baremetal/nodes-custom-ip.md:10]: This resolver is used in IPI case - is it being run in UPI too?

Comment (GingerGeek) [reply Guides/UPI/baremetal/nodes-custom-ip.md:10]: I think so

Comment (GingerGeek) [reply Guides/UPI/baremetal/nodes-custom-ip.md:10]: Yes, I'm on bare metal UPI and it's being ran

**Review by vrutkovs — APPROVED** [review]
/lgtm

**Pull request merged**